packages/core/package.json

```diff
@@ -7,7 +7,7 @@
   "author": "jaketrent",
   "keywords": [],
   "main": "dist/index.js",
-  "module": "js/index.js",
+  "module": "js/index.ts",
```

Comment:
can you use `ts` as a module. what about 'dist/index.js'

Reply:
nope, can't. I was hoping, but then I tried it. :)  dist/index.js is cjs, so no go there.  We really should just remove this module line, because I don't think we generate a true module in core. Well, better, I guess would be to gen it.  Maybe I'll do that.